pkg/gui/controllers/commits_files_controller.go

```diff
@@ -66,10 +69,9 @@ func (self *CommitFilesController) GetKeybindings(opts types.KeybindingsOpts) []
 			Description:       self.c.Tr.OpenDiffTool,
 		},
 		{
-			Key:               opts.GetKey(opts.Config.Universal.Select),
-			Handler:           self.withItem(self.toggleForPatch),
-			GetDisabledReason: self.require(self.singleItemSelected()),
```

Comment:
`GetDisabledReason` is still needed here i.e. 'itemsSelected'

Reply:
Got it, added this in